README.md

```diff
@@ -16,7 +16,7 @@ This CDK library automatically configures ingestion of metrics, traces, and logs
 
 ## AWS CDK v1 vs AWS CDK v2
 
-**WARNING**:  `AWS CDK v1` has reached end-of-support and `datadog-cdk-constructs` will no longer be receiving updates. It's strongly recommended to upgrade to `AWS CDK v2` ([official migration guide](https://docs.aws.amazon.com/cdk/v2/guide/migrating-v2.html)) and switch to using `datadog-cdk-constructs-v2`.
+**WARNING**: `AWS CDK v1` has reached end-of-support and `datadog-cdk-constructs` will no longer be receiving updates. It's strongly recommended to upgrade to `AWS CDK v2` ([official migration guide](https://docs.aws.amazon.com/cdk/v2/guide/migrating-v2.html)) and switch to using `datadog-cdk-constructs-v2`.
```

Comment:
Lots of changes on this file was formatting changes automatically done by VS Code. Let me you if you think I should revert them.